### PR TITLE
Escenarios 

### PR DESCRIPTION
**Pull request by EriGoldenberg**

### CODE REVIEW

File: src/test/resources/unrc/dose/Goldenberg.feature [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      public static int module(int a){
+        if(a >= 0){
+            return -a;
+        }
+        else{
+          return a;
+        }
+      }
+    }
+  ”
```
Comment (saiema) [inline src/test/resources/unrc/dose/Goldenberg.feature]: I think that we should use triple tilde when using multi line strings in cucumber syntax ()

File: src/test/resources/unrc/dose/Goldenberg.feature [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  Given  the user “Roberta” is already logged on
+  And the user is a content creator user
+  And the text of the challenge is:
+    “
+    Public class Max{
+      Public static int maxi(int n, int m){
+        if(n>=m){
+          Return n;
+        }
+        Else{
```
Comment (saiema) [inline src/test/resources/unrc/dose/Goldenberg.feature]: I think that we should parse correctly the java syntax here

**Comment by boaglio15** [comment]
Closing invalid

**Comment by nmaguirre** [comment]
Closing this pull request. It's superseeded by a newer one.

**Pull request closed without merging**